src/material/select/select.html

```diff
@@ -13,7 +13,7 @@
      (click)="toggle()"
      #origin="cdkOverlayOrigin"
      #trigger>
-  <div class="mat-select-value" [ngSwitch]="empty" [attr.id]="_valueId">
+  <div class="mat-select-value" [ngSwitch]="empty" [attr.id]="_valueId" [attr.aria-hidden]="true">
```

Comment:
This can be reduced to `aria-hidden="true"`. Also you'll have to make the same change in the MDC-based select.